__tests__/input-helper.test.ts

```diff
@@ -178,27 +178,101 @@ describe('input-helper tests', () => {
     }
     inputs.pattern = 'some-pattern'
     inputs.error = 'some-error'
-    inputs.excludeDescription = '1'
-    const checkerArguments: ICheckerArguments = inputHelper.getInputs()
+    inputs.excludeDescription = 'true'
+    const checkerArguments: ICheckerArguments = await inputHelper.getInputs()
     expect(checkerArguments).toBeTruthy()
     expect(checkerArguments.pattern).toBe('some-pattern')
     expect(checkerArguments.error).toBe('some-error')
     expect(checkerArguments.messages).toBeTruthy()
     expect(checkerArguments.messages[0]).toBe('some-title')
   })
 
-  it('push payload is optional', () => {
+  it('excludes pull_request title', async () => {
+    mockGitHub.context = {
+      eventName: 'pull_request',
+      payload: {
+        pull_request: {
+          title: 'some-title',
+          body: 'some-body'
+        }
+      }
+    }
+    inputs.pattern = 'some-pattern'
+    inputs.error = 'some-error'
+    inputs.excludeTitle = 'true'
+    const checkerArguments: ICheckerArguments = await inputHelper.getInputs()
+    expect(checkerArguments).toBeTruthy()
+    expect(checkerArguments.pattern).toBe('some-pattern')
+    expect(checkerArguments.error).toBe('some-error')
+    expect(checkerArguments.messages).toBeTruthy()
+    expect(checkerArguments.messages[0]).toBe('some-body')
+  })
+
+  it('excludes pull_request title and body', async () => {
+    mockGitHub.context = {
+      eventName: 'pull_request',
+      payload: {
+        pull_request: {
+          title: 'some-title',
+          body: 'some-body'
+        }
+      }
+    }
+    inputs.pattern = 'some-pattern'
+    inputs.error = 'some-error'
+    inputs.excludeDescription = 'true'
+    inputs.excludeTitle = 'true'
+    const checkerArguments: ICheckerArguments = await inputHelper.getInputs()
+    expect(checkerArguments).toBeTruthy()
+    expect(checkerArguments.pattern).toBe('some-pattern')
+    expect(checkerArguments.error).toBe('some-error')
+    expect(checkerArguments.messages).toBeTruthy()
+    expect(checkerArguments.messages.length).toBe(0)
+  })
+
+  it('should check pull_request commits', async () => {
+    mockGitHub.context = {
+      eventName: 'pull_request',
+      payload: {
+        pull_request: {
+          title: 'some-title',
+          body: 'some-body',
+          number: 62 // This pull request has exactly two commits
```

Comment:
Where is this numer `62`coming from?

Reply:
this is the pull request number [#62](https://github.com/TotalCross/totalcross/pull/62) from TotalCross. I've chosen a real case scenario to test if the commits were coming.